pystiche_papers/sanakoyeu_et_al_2018/_modules.py

```diff
@@ -273,3 +276,45 @@ def transformer(style: Optional[str] = None,) -> Transformer:
 
     """
     return Transformer()
+
+
+class Discriminator(pystiche.Module):
+    r"""Discriminator from :cite:`SKL+2018`.
+
+    Args:
+        in_channels: Number of channels in the input. Defaults to ``3``.
+    """
+
+    def __init__(self, in_channels: int = 3) -> None:
+        def discriminator_encoder_modules(
```

Comment:
What is the point of this function? it is only used once and thus the functionality could be simply inlined.